doc/developer-notes.md

```diff
@@ -834,8 +836,25 @@ GUI
   - *Rationale*: Blocking the GUI thread can increase latency, and lead to
     hangs and deadlocks.
 
-Subtrees
-----------
+Git Usage
+---------
+
+### Rebasing
+
+When the master development branch develops conflicts with your pull request,
+you need to rebase it. This means rebuilding your branch on top of a new base.
```

Comment:
"rebuilding" here is ambiguous. I think you mean "apply the commits in your branch to the master branch", but it could also be interpreted as meaning "compiling and linking"